mocked_servers/Makefile

```diff
@@ -0,0 +1,17 @@
+ADDLICENSE=$(PWD)/bin/addlicense
+
+.PHONY: install-tools
+install-tools:
```

Comment:
Think it's better to use the OTel docker build tool

https://github.com/open-telemetry/build-tools/blob/main/protobuf/README.md

Reply:
Ended up just using https://github.com/open-telemetry/opentelemetry-proto-go instead as suggested by @Aneurysm9